Application/EdFi.Ods.Features/Controllers/DeletesController.cs

```diff
@@ -19,7 +19,7 @@ namespace EdFi.Ods.Features.Controllers
     [Authorize]
     [ApiController]
     [Produces("application/json")]
-    [Route("{schema}/{resource}/deletes")]
+    [Route("{schema}/{resource}/deletes", Order = -1)]
```

Comment:
It's recommended that avoid using order here [docs.microsoft.com/en-us/aspnet/core/mvc/controllers/routing?view=aspnetcore-6.0](https://docs.microsoft.com/en-us/aspnet/core/mvc/controllers/routing?view=aspnetcore-6.0)

Reply:
After further discussions with the team, to get tests passing so further changes can be made to get everything on NET6, this fix will stay in place for now and a subsequent ticket is going to be added for a more permanent fix to handle the [route order/precedence changes that happened as a part of the upgrade](https://github.com/dotnet/aspnetcore/issues/29594#issuecomment-767063089) . One option being looked at is moving the deletes into CodeGen to be part of each aggregate instead of the catch all we have here. But the future ticket will handle seeing if that is a viable option or if we need to handle it a different way.